mvnw

```diff
@@ -0,0 +1,225 @@
+#!/bin/sh
```

Comment:
I think it would be important to document what these files are for, maybe in the top Onboarding Guide wiki as well?